### ISSUE TITLE
Release FasterPy artifacts (models, datasets) on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [WuYue22/FasterPy#1]
Hi @WuYue22 🤗

Niels here from the open-source team at Hugging Face. I discovered your work on Arxiv and was wondering whether you would like to submit it to [hf.co/papers](https://hf.co/papers) to improve its discoverability. If you are one of the authors, you can submit it at https://huggingface.co/papers/submit.

The paper page lets people discuss your work and find related artifacts (your models, datasets, or demo for instance). You can also claim the paper as yours, which will make it show up on your public profile at HF, and you can add Github and project page URLs.

It's great to see that you've already hosted the **OD-base** knowledge base on the Hub! It'd be awesome to also make the training and evaluation datasets available on the 🤗 hub to improve their discoverability and visibility.

## Uploading models

Would you also like to host the model weights you've fine-tuned (such as the LoRA adapters for Qwen2.5 or DeepSeek-Coder) on https://huggingface.co/models? Hosting on Hugging Face will give your work more visibility. See here for a guide: https://huggingface.co/docs/hub/models-uploading. 

For LoRA adapters, users can easily load them alongside base models. Alternatively, one can leverage the [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file) one-liner to download a checkpoint from the hub.

## Uploading dataset

Hosting the remaining datasets (like `train_data_msg.jsonl` and the improved PIE test set) on 🤗 would allow people to do:

```python
from datasets import load_dataset

dataset = load_dataset("your-hf-org-or-username/your-dataset")
```
See here for a guide: https://huggingface.co/docs/datasets/loading.

Besides that, there's the [dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer) which allows people to quickly explore the first few rows of the data in the browser.

Let me know if you're interested or need any help regarding this!

Cheers,

Niels
ML Engineer @ HF 🤗
